workspace-manager-clienttests/README.md

```diff
@@ -38,13 +47,13 @@ The version of the Workspace Manager client JAR file is specified in the build.g
 fetched from the Broad Institute Maven repository. You can override this to use a local version of the Workspace Manager client
 JAR file by specifying a Gradle project property, either with a command line argument
 
-`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="configs/BasicUnauthenticated.json"
+`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="suites/BasicIntegration.json"
```

Comment:
I think the single test config run command is more helpful. Or include both. But just the suite run command is probably not sufficient for all use cases.

Reply:
I have added both test modes: single test and test suite in separate sections.
Developers should be able to follow the guidelines to create their own use cases, whether for individual tests or grouping multiple tests into a test suite.